run.rb

```diff
@@ -0,0 +1,61 @@
+require_relative './app'
+require_relative './book'
+
+class Run
+  include App
+  include RentalListing
+  include BookLisitng
+  include PeopleListing
+
+  def initialize
+    @books = []
+    @peoples = []
+    @rentals = []
+  end
+
+  def main
+    loop do
+      puts "\n\nWelcome to School Library APP! \n\n"
+      print 'Please Choose an option by enerin a number: '
+      puts "
+        1 - List all books
+        2 - List all people
+        3 - Create a person
+        4 - Create a book
+        5 - Create a rental
+        6 - List all rentals for a given person id
+        7 - Exit
+        "
+      print 'option: '
+      option = gets.chomp
+      show(option)
+    end
+  end
+
+  def show(option)
+    case option
+    when '1'
+      list_all_books(@books)
+    when '2'
+      list_all_peoples(@peoples)
+    when '6'
+      list_rentals(@rentals, @peoples)
+    when '7'
+      puts 'Exit'
+      exit
+    else
+      create(option)
+    end
+  end
+
+  def create(option)
+    case option
+    when '3'
+      create_person(@peoples)
```

Comment:
- [ ] Kindly refactor the `create_person`, `create_teacher`, `create_student`, `create_book`, and `create_rental` methods. In these methods, all codes responsible for **_taking input and validating inputs_** should be in a separate method, module or class while these methods should be responsible for creating the **_class instances_**. Kindly refactor the code. 👍 💻

Reply:
@Gambit142 ,  Ok, I will refactor to classes this methods and separate in modules.
Thank you